WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -365,6 +365,14 @@ class MeViewController: UITableViewController {
         navigateToTarget(for: RowTitles.accountSettings)
     }
 
+    /// Selects the All Domains row and pushes the All Domains view controller
+    ///
+    public func navigateToAllDomains() {
+    #if JETPACK
```

Comment:
Adding this to the function itself will cause us to add more `if Jetpack` checks down the chain. I'd rather go it this way but if we prefer to do it properly, I can add those calls. LMK